### ISSUE TITLE
Migration of methods

### ISSUE THREAD

**Issue by miili** [pydantic/bump-pydantic#49]
Hi all,

when is method-migration planned? I saw it on the roadmap. Has it been put down?

M

**Comment by samuelcolvin** [comment]
hi @miili, I'm afraid I don't quite understand your question?

**Comment by Kludex** [comment]
You mean `dict` to `model_dump`, etc.?

**Comment by miili** [comment]
> You mean `dict` to `model_dump`, etc.?

Yes, automatic method migration as to https://docs.pydantic.dev/dev-v2/migration/#changes-to-pydanticbasemodel

**Comment by Kludex** [comment]
I'm going to let this issue open, but I don't think we'll be supporting it.

The effort to be accurate on these are huge, and currently, libcst only supports pyre for type inference, which is not that easy to set up.

What I mean is that, I need to make sure that an object is a BaseModel, and that's not trivial.

**Comment by Kludex** [comment]
> I've found a way. I'll be adding them.

I had to revert the way I found... The way to do this is either using `astroid` or `TypeInference` from LibCST.
